.eslintrc.js

```diff
@@ -23,10 +23,13 @@ module.exports = {
     'BROWSERSTACK_USERNAME': false,
     'BROWSERSTACK_KEY': false
   },
+  // use babel as parser for fancy syntax
+  parser: '@babel/eslint-parser',
```

Comment:
Why do we need `@babel/eslint-parser`?  
Looking at the npm description I see:

> ESLint's default parser and core rules only support the latest final ECMAScript standard and do not support experimental (such as new features) and non-standard (such as Flow or TypeScript types) syntax provided by Babel

Do we need these 'experimental and non-standard' syntax? Or am I missing something?

Reply:
It's part of the `class` syntax, specifically field declaration and private (#) methods and fields. Btw I care about objects but not really about syntax, I don't mind going either way with this.